R/Angel Chen/ticket #20492 Norman.R

```diff
@@ -0,0 +1,144 @@
+#ticket #20492 Norman
+#Human dataset:
+#https://arcticdata.io/catalog/#view/urn:uuid:c715fe12-6c05-4c27-8d00-b9c0c536c54b
+
+pkg <- get_package(adc, 'resource_map_urn:uuid:2a957ad3-1c0f-44e9-b79d-5ea2a2ed76bf', file_names = TRUE)
+doc <- read_eml(getObject(adc, pkg$metadata))
+emld::eml_version("eml-2.1.1")
```

Comment:
hopefully this will be something we won't need to think about as much moving forward with everything in 2.2.0 but yeah we need to keep this in mind when working with older datasets